pom.xml

```diff
@@ -111,6 +111,11 @@
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>edge-common</artifactId>
+      <version>0.3.3-SNAPSHOT</version>
+    </dependency>
```

Comment:
Why do we need a dependency on edge-common?

Reply:
Yeah, I should remove it. I thought I needed it to extract the okapi token.